go-controller/pkg/util/pod_annotation.go

```diff
@@ -191,6 +198,11 @@ func MarshalPodAnnotation(annotations map[string]string, podInfo *PodAnnotation,
 			NextHop: nh,
 		})
 	}
+
+	if podInfo.IPv6LLAGateway != nil {
```

Comment:
Validate its a valid ip before setting it ?

Reply:
no need imho, all validations are already done at the setter
see please https://github.com/ovn-kubernetes/ovn-kubernetes/pull/4878